supertokens_python/recipe/session/interfaces.py

```diff
@@ -34,7 +34,9 @@
 from .utils import SessionConfig, TokenTransferMethod
 
 if TYPE_CHECKING:
-    from supertokens_python.framework import BaseRequest, BaseResponse
```

Comment:
This is strange: nothing changed below, so it either didn't work before or is not necessary now. So I'm happy to close this; I just want to understand what this improves.

Reply:
`TYPE_CHECKING` is true only when you run `pylint`. But not when you're running the code with `python`. 
There are the error wasn't spotted by `pylint` because `TYPE_CHECKING` was True and import was successful.

But when running with python, `BaseResponse` was not defined (since `TYPE_CHECKING  = False`). So python threw error at ResponseMutator definition since it needs `BaseResponse` to be defined.